ui/src/tool-panel/km-post/dialog/km-post-edit-store.ts

```diff
@@ -91,30 +126,65 @@ function getErrorForKmPostExistsOnTrack(): FieldValidationIssue<KmPostSaveReques
     ];
 }
 
+function getGkCoordinateDoesNotParseError(
+    field: keyof KmPostEditFields,
+): FieldValidationIssue<KmPostEditFields>[] {
+    return [
+        {
+            field,
+            reason: 'gk-coordinate-parse',
+            type: FieldValidationIssueType.ERROR,
+        },
+    ];
+}
+
+function getGkCoordinateIsNotFaithfullySaveableError(
+    field: keyof KmPostEditFields,
+): FieldValidationIssue<KmPostEditFields>[] {
+    return [
+        {
+            field,
+            reason: 'gk-coordinate-saveable-as-float',
```

Comment:
Tähän voisi lyödä "not"-sanan johonkin väliin. Nykyisestä jää vähän sellainen fiilis että ongelma olisi että koordinaatti _on_ tallennettavissa liukulukuna